systemtest/driver/inifile.py

```diff
@@ -37,3 +38,27 @@ def instanciate_tempalte(*, src: pathlib.Path, args: Dict, dest: pathlib.Path):
     tpl = jinja2.Template(src.read_text())
     content = tpl.render(args)
     dest.write_text(content)
+
+
+class TrafficConstraint(NamedTuple):
+    door_id: int
+    outflow: int
+    max_agents: int
```

Comment:
I have two remarks about this code:

You could declare this as a dataclass which has similar semantics but is not a `NamedTuple` in case of `instanceof` calls.

We can stop using '-1' as indicator for "I have no value" and instead use 'None' instead. Additionally we can clearly communicate this by using `typing.Optional(int)`

E.g.:
```
from dataclasses import dataclass
from typing import Optional

@dataclass
class TrafficConstraint:
    door_id: int
    outflow: Optional(int) # <- this tells the type hints that 'None' is also acceptable
    max_agents: Optional(int)
```

Reply:
done